app/src/js/reducers/rules.js

```diff
@@ -59,9 +59,8 @@ export default function reducer (state = initialState, action) {
   const { data, id } = action;
   switch (action.type) {
     case RULE:
-      const foundRule = data.results.find((result) => result.name === id);
```

Comment:
This was only necessary because the rules action was previously querying the GET endpoint listing all rules rather than using the GET endpoint for a specific rule. That is also what was causing there to be an extra `timestamp` property on the `rule` which also was causing PUT updates to fail.